api/helpers.go

```diff
@@ -140,13 +140,17 @@ func (st *DCOSTools) GetUnitNames() (units []string, err error) {
 func (st *DCOSTools) GetJournalOutput(unit string) (string, error) {
 	matches := []reader.JournalEntryMatch{
 		{
+			Field: "_SYSTEMD_UNIT",
```

Comment:
These slices of `reader.JournalEntryMatch` are duplicated between `GetJournalOutput` and `readJournalOutputSince` -- does it make sense to save these as a package `var` ?